backend/app/controllers/internal/login_controller.rb

```diff
@@ -1,9 +1,7 @@
 # frozen_string_literal: true
 
-class Api::V1::LoginController < Api::BaseController
-  include OtpValidation, UserDataSerialization
-
-  skip_before_action :authenticate_with_jwt
+class Internal::LoginController < Internal::BaseController
+  include OtpValidation, UserDataSerialization, JwtAuthenticatable
```

Comment:
We don't need `JwtAuthenticatable` here since the token wouldn't be present / won't be used?

```suggestion
  include OtpValidation, UserDataSerialization
```